packages/react-instantsearch-dom/src/components/SearchBox.js

```diff
@@ -249,6 +264,7 @@ class SearchBox extends Component {
     return (
       <div className={classNames(cx(''), className)}>
         <form
+          ref={this.onFormMount}
```

Comment:
```suggestion
          ref={this.props.formRef}
```

as this.form isn't used, there's no need for the function dance I think

Reply:
Yes, that makes sense. I just mimicked `onInputMount`, but there was no real need for that. Thanks!